src/pages/library/Datasets.js

```diff
@@ -632,24 +643,31 @@ export const Datasets = () => {
     }),
     catalogShowing
       ? h(Browser)
-      : div({ role: 'main', style: styles.content }, [
-          // Put datasets in alphabetical order
-          thousandGenomesHighCoverage(),
-          thousandGenomesLowCoverage(),
-          amppd(),
-          baseline(),
-          ccdg(),
-          cmg(),
-          encode(),
-          fcDataLib(),
-          framingham(),
-          gp2(),
-          hca(),
-          nemo(),
-          rareX(),
-          target(),
-          tcga(),
-          topMed(),
+      : div([
```

Comment:
I think this change should wait until we add the axin dataset as a card on this page. Also I see in the mocks that the new page is supposed to change the way the new catalog view is handled, and add a card for anvil.

If we do want to change this page in this PR, we first need a mock that reflects what's in the PR, rather than what will be in a future PR.

Reply:
Okay I can revert this change